src/jira/client/axios.ts

```diff
@@ -55,15 +55,13 @@ function getAuthMiddleware(secret: string) {
 	);
 }
 
-/**
- * Mapping of Jira Dev Error Codes to nice strings.
- */
-const JiraErrorCodes = {
-	400: "Request had incorrect format.",
-	401: "Missing a JWT token, or token is invalid.",
-	403: "The JWT token used does not correspond to an app that defines the jiraDevelopmentTool module, or the app does not define the 'WRITE' scope",
-	413: "Data is too large. Submit fewer devinfo entities in each payload.",
-	429: "API rate limit has been exceeded."
+export const JiraErrorMessages = {
+	400: (detailMessage) => `HTTP 400 - Request had incorrect format. Details: ${detailMessage}`,
+	401: () => "HTTP 401 - Missing a JWT token, or token is invalid.",
+	403: () => "HTTP 403 - The JWT token used does not correspond to an app that defines the jiraDevelopmentTool module, or the app does not define the 'WRITE' scope",
+	413: () => "HTTP 413 - Data is too large. Submit fewer devinfo entities in each payload.",
+	429: () => "HTTP 429 - API rate limit has been exceeded.",
+	"default": (status, detailMessage) => `HTTP ${status} - ${detailMessage}`
```

Comment:
I collected ALL errors in this single place now to make it more obvious what's happening